specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2021-05-01/managedClusters.json

```diff
@@ -1776,8 +1752,8 @@
           "ManagedClusters"
         ],
         "operationId": "ManagedClusters_RunCommand",
-        "summary": "Run Command against Managed Kubernetes Service",
-        "description": "Submit a command to run against managed kubernetes service, it will create a pod to run the command.",
+        "summary": "Submits a command to run against the Managed Cluster.",
+        "description": "AKS will create a pod to run the command. This is primarily useful for private clusters. For more information see [AKS Run Command](https://docs.microsoft.com/azure/aks/private-clusters#aks-run-command-preview).",
```

Comment:
The docs make this sound like it's primarily for private clusters. Does it work on non-private clusters?

Reply:
Believe it does Haitao would know?

Reply:
It does work on non-private clusters.